src/expression.ts

```diff
@@ -156,6 +156,22 @@ export class BinaryExpression extends Expression {
     }
 }
 
+/** it uses to specified a wasm function call in compile time,
+ * sometimes the wasm function maybe is not generated from a ts function
+ */
+export class EnumerateKeysExpression extends Expression {
+    private obj: Expression;
+
+    constructor(obj: Expression) {
+        super(ts.SyntaxKind.PropertyAccessExpression);
```

Comment:
Why we use `PropertyAccessExpression` here?

Reply:
In `ts.SyntaxKind` seems to lack an appropriate type to uniquely represent the access of enumerate keys. Therefore, here we use `ts.SyntaxKind.PropertyAccessExpression` to indicate obtaining the key attribute from an object.

Reply:
Got it, had better leave a comment here

Reply:
Would it be better to use `SyntaxKind.Unknown` to indicate this is a customized node?

Reply:
I don't think it's very suitable, because `SyntaxKind.Unknown` has its own meaning in the frontend, it has already been used to represent some nodes that the compiler doesn't support or implement.

Reply:
OK